src/PortfolioPlanning/Components/Plan/DependencyPanel.tsx

```diff
@@ -103,6 +102,10 @@ export class DependencyPanel extends React.Component<IDependencyPanelProps, IDep
         } else if (this.state.loading != LoadingStatus.Loaded) {
             return <Spinner label="Loading dependencies..." size={SpinnerSize.large} className="loading-spinner" />;
         } else {
+            //Sort dependencies by target date
+            this.state.dependsOn.sort((a, b) => (a.TargetDate > b.TargetDate) ? 1 : -1);

```

Comment:
We do this sorting when they're fetched once, rather than on each render.

Reply:
Moved sorting inside the constructor.